dataactvalidator/filestreaming/loadFile.py

```diff
@@ -1,53 +1,216 @@
     loadObjectClass(os.path.join(basePath,"object_class.csv"))
+    print("Loading program activity")
+    if localProgramActivity is not None:
+        loadProgramActivity(localProgramActivity)
+    else:
+        loadProgramActivity(os.path.join(basePath, "program_activity.csv"))
 
     # SF 133 is kept on S3, so need to download that
     reader = CsvS3Reader()
 
     if localSFPath is not None:
-        # Load SF 133 from same path
         print("Loading local SF-133")
-        loadSF133(localSFPath)
+        # get list of SF 133 files in the specified local directory
+        sf133Files = glob.glob(os.path.join(localSFPath, 'sf_133*.csv'))
+        for sf133 in sf133Files:
+            file = os.path.basename(sf133).replace('.csv', '')
+            fileParts = file.split('_')
+            if len(fileParts) < 4:
+                print('{}Skipping SF 133 file with invalid name: {}'.format(
+                    os.linesep, sf133))
+                continue
+            year = file.split('_')[-2]
+            period = file.split('_')[-1]
+            print('{}Starting {}...'.format(os.linesep, sf133))
+            loadSF133(sf133, year, period)
     else:
-        # Download files if using aws, if not they will need to already be in config folder
-        print("Loading default SF-133")
+        print("Loading SF-133")
         if(CONFIG_BROKER["use_aws"]):
-            reader.downloadFile(CONFIG_BROKER["aws_region"],CONFIG_BROKER["aws_bucket"],"/".join([CONFIG_BROKER["sf_133_folder"],CONFIG_BROKER["sf_133_file"]]),os.path.join(CONFIG_BROKER["path"],"dataactvalidator","config",CONFIG_BROKER["sf_133_file"]))
-
-        loadSF133(os.path.join(CONFIG_BROKER["path"],"dataactvalidator","config",CONFIG_BROKER["sf_133_file"]))
-    print("Loading program activity")
-    if localProgramActivity is not None:
-        loadProgramActivity(localProgramActivity)
-    else:
-        loadProgramActivity(os.path.join(basePath,"program_activity.csv"))
-
+            # get list of SF 133 files in the config bucket on S3
+            s3connection = boto.s3.connect_to_region(CONFIG_BROKER['aws_region'])
+            s3bucket = s3connection.lookup(CONFIG_BROKER['aws_bucket'])
+            # get bucketlistresultset with all sf_133 files
+            sf133Files = s3bucket.list(
+                prefix='{}/sf_133'.format(CONFIG_BROKER['sf_133_folder']))
+            for sf133 in sf133Files:
+                file = sf133.name.split(
+                    CONFIG_BROKER['sf_133_folder'])[-1].replace('.csv', '')
+                fileParts = file.split('_')
+                if len(fileParts) < 4:
+                    print('{}Skipping SF 133 file with invalid name: {}'.format(
+                        os.linesep, sf133))
+                    continue
+                year = file.split('_')[-2]
+                period = file.split('_')[-1]
+                print('{}Starting {}...'.format(os.linesep,sf133.name))
+                loadSF133(sf133, year, period)
```

Comment:
This is _very_ similar to the `localSFPath is not None` path above. Can they be combined?